core/plugins/org.fusesource.ide.camel.model.service.impl/src/org/fusesource/ide/camel/model/service/internal/CamelModelPatcher.java

```diff
@@ -164,6 +164,9 @@ private static void applyFixesToComponentsSyntax(CamelModel loadedModel) {
 		applyFixToComponent(loadedModel, "micrometer", "micrometer:metricsType:meterName", "micrometer:metricsType:metricsName");
 		applyFixToComponent(loadedModel, "service", "service:serviceName:delegateUri", "service:delegateUri");
 		applyFixToComponent(loadedModel, "nsq", "nsq:lookupServer", "nsq:servers");
+		applyFixToComponent(loadedModel, "corda", "corda:url", "corda:configuration");
```

Comment:
thsi is not related to 7.3.1 version but to 2.24 Camel catalog. Would be really best to have it in a different commit/PR